booking-engine-api/operations/availability-blocks.md

```diff
@@ -0,0 +1,66 @@
+# Availability blocks
+
+## Get Availability blocks
+
+Get availability block details for specified booking engine instance, that modifies your availability of space categories, and rates.
+This operation can be called initially to fetch data which may be important during the booking workflow.
+Availability block can restrict your booking engine's calendar to specific interval defined by `StartUtc` and `EndUtc` in the response, it also gives you `RateId` that should be used.
+
+### Request
+
+`[ApiBaseUrl]/api/distributor/v1/availabilityBlocks/getAll`
```

Comment:
It may be a small point, but strictly the endpoint is 'getAll' yet the name of the operation is 'Get availability blocks' not 'Get all availability blocks', and in any case you can't simply get all availability blocks, you have to specify the IDs of the availability blocks you want to fetch, so it's misnamed.

Reply:
Yes, `AvailabilityBlockIds` argument is required to get the list of blocks. It doesn't work that it gives you all.